sdk/cmake/msvc.cmake

```diff
@@ -321,13 +321,11 @@ function(generate_import_lib _libname _dllname _spec_file)
     if(MSVC_IDE)
         add_library(${_libname} STATIC EXCLUDE_FROM_ALL ${_asm_stubs_file}.obj)
         set_source_files_properties(${_asm_stubs_file}.obj PROPERTIES EXTERNAL_OBJECT TRUE)
-        set_target_properties(${_libname} PROPERTIES LINKER_LANGUAGE "C")
     else()
         # NOTE: as stub file and def file are generated in one pass, depending on one is like depending on the other
         add_library(${_libname} STATIC EXCLUDE_FROM_ALL ${_asm_stubs_file})
-        # set correct "link rule"
-        set_target_properties(${_libname} PROPERTIES LINKER_LANGUAGE "IMPLIB")
     endif()
+    set_target_properties(${_libname} PROPERTIES LINKER_LANGUAGE "C")
```

Comment:
This will make clang-cl on GitHub Actions unhappy. It defaults to using `llvm-lib` with the latest CMake. We have an explicit override to use `link /lib` for IMPLIB linking, which was how I fixed this.

With this, we may need to override `CMAKE_C_CREATE_STATIC_LIBRARY` on Clang-CL (or all MSVC) builds.